homeassistant/components/filesize/strings.json

```diff
@@ -0,0 +1,18 @@
+{
+    "config": {
```

Comment:
`Filesize` could be translated perfectly and thus should be in the translations file.